fern/apis/image-gen/openapi/openapi-overrides.yml

```diff
@@ -44,6 +44,13 @@ paths:
       servers:
         - url: https://image.octoai.run
           x-name: ImageGen
+  /generate/ip-adapter-sdxl:
+    post:
+      summary: "IP Adapter"
+      x-fern-sdk-method-name: ip_adapter
+      servers:
+        - url: https://image.octoai.run
+          x-name: ImageGen
```

Comment:
I don't think we need this anymore. I had them put the 'servers' in the specs

Reply:
I never had any luck with root level `servers` declarations in the overrides, but I can try it and see if it works.

Reply:
I'm going to create a separate PR to attempt removing all the `servers` so we don't keep copy/pasting.

Reply:
Doesn't work https://github.com/octoml/fern-config/pull/78.  May need to ask fern why.